sdk/servicebus/azure-servicebus/README.md

```diff
@@ -265,7 +265,7 @@ with ServiceBusClient.from_connection_string(connstr) as client:
 When receiving from a queue, you have multiple actions you can take on the messages you receive.
 
 > **NOTE**: You can only settle `ReceivedMessage` objects which are received in `ReceiveSettleMode.PeekLock` mode (this is the default).
-> `ReceiveSettleMode.ReceiveAndDelete` mode removes the message from the queue on receipt.  `PeekMessage` messages
+> `ReceiveSettleMode.ReceiveAndDelete` mode removes the message from the queue on receipt.  `PeekedMessage` messages
```

Comment:
we probably want to do a replace all "ReceiveSettleMode" with "ReceiveMode"

Reply:
I did that but had actually unnecessarily limited my search to exclude doc files :x  Good catch, will make sure this is covered.

Reply:
(Looks like this is the only one that slipped through, entertainingly)